docs/source/concepts/dataset.md

```diff
@@ -28,6 +28,15 @@ Supports `[.png, .jpg, .jpeg]` file extensions.
                 - ...
             - ...
 
+#### Tabular Data
+Supports `.csv` file formats.
+
```

Comment:
right here, you should add at least 1 notebook. if you look at the `Computer Vision` section above, i have a table with a notebook. could you please copy that notebook & use one of the kaggle datasets you were using in the beginning?

Reply:
here is the example notebook i was referring to: https://colab.research.google.com/drive/19Qb5LP-TP0CEn8_IRsrJkbZK6OubqPcq?usp=sharing